Doc/reference/expressions.rst

```diff
@@ -1342,7 +1342,7 @@ returned; otherwise, *y* is evaluated and the resulting value is returned.
 The expression ``x or y`` first evaluates *x*; if *x* is true, its value is
 returned; otherwise, *y* is evaluated and the resulting value is returned.
 
-(Note that neither :keyword:`and` nor :keyword:`or` restrict the value and type
+Note that neither :keyword:`and` nor :keyword:`or` restrict the value and type
```

Comment:
There is a closing parenthesis at the end of the paragraph.

Reply:
Good catch!

Reply:
There's no unbalanced parenthesis in the current version for 2.7 (since the example given at the end is not between parenthesis). Should I leave this particular file as it is now? That is, undoing the change in this PR.

Reply:
I think that if there is nothing wrong, nothing should be changed. It is better to leave this particular file as it is now.

Reply:
I agree.